examples/Polls API.md

```diff
@@ -0,0 +1,179 @@
+FORMAT: 1A
+
+# Polls
+
+Polls is a simple web service that allows consumers to view polls and vote in them.
+
+# Polls API Root [/]
+
+This resource does not have any attributes. Instead it offers the initial API affordances in the form of the links in the JSON body.
+
+It is recommend to follow the “url” link values or Link headers to get to resources instead of constructing your own URLs to keep your client decoupled from implementation details.
```

Comment:
Yeah, it might be too much information. But how about just 

> It is recommend to follow the “url” link values or Link or Location headers where applicable ...

?